src/broker/broker.py

```diff
@@ -168,7 +168,7 @@ def publish_message(self, producer_id: int, topic_name: str, message: str):
                 }]
             }
         self.__enqueue_logs.append("INSERT INTO topic(topic_name, partition_id,bias) SELECT '" + topic_name + "','" + str(self.broker_id) +
-                                   " ', '0' WHERE NOT EXISTS (SELECT topic_name, partition_id FROM topic WHERE topic_name = '" + topic_name + "' and partition_id =" + str(self.broker_id) + ");")
+                                   "', '0' WHERE NOT EXISTS (SELECT topic_name, partition_id FROM topic WHERE topic_name = '" + topic_name + "' and partition_id =" + str(self.broker_id) + ");")
```

Comment:
This triggered an error "if" condition, which took 1 day for root cause analysis.

Assumed something must be wrong with multithreading and "thread-unsafe" operations causing inconsistent dictionaries but nope, an extra space was the root cause.